guardian/cloudwatch-logs-management
# Upgrade to Node 14

## What does this change?
Upgrades the project to use Node 14, as Node 10 reaches EOL soon.

## How to test
I have [deployed this change to the Deploy Tools account](https://riffraff.gutools.co.uk/deployment/view/312d53e6-addd-4902-98a6-86634dc99c57). I will check lambda logs / metrics and confirm that logs from other DevX-owned lambdas still turn up in Central ELK before merging this change.

## How can we measure success?
We will be using a version of Node which is properly supported after 30/04/2021!

## Have we considered potential risks?
Yes, we are upgrading several major versions in one move, so there is some risk involved! In order to mitigate this risk I'm testing this change by deploying it to a single account only (once merged this upgrade will affect many lambdas in [multiple accounts](https://github.com/guardian/cloudwatch-logs-management/blob/0263253ca78699cdefaf64e87682706b28065046/riff-raff.yaml#L3-L14)).
